Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -126,6 +126,9 @@ public async Task InitAsync(PluginInitContext context)
 
             if (!(_win32s.Any() && _uwps.Any()))
                 await indexTask;
+            
+            Win32.WatchProgramUpdate(_settings);
```

Comment:
do you not also need to add WatchProgramUpdate for UWP apps?

Reply:
Huh I forget? I will check soon.

Reply:
Ah no. UWP are watched by package update